workshop/01_Navigation/05_Find_usages/5.2_Find_Usages.php

```diff
@@ -6,10 +6,13 @@
  *
  * Show Usages (uses popup)
  *
- * Ctrl+Alt+F7 (Windows/Linux)
+ * Ctrl+Alt+7 (Windows/Linux)
```

Comment:
Please leave the default shortcut as it is.

Reply:
This was the default shortcut for Ubuntu.